include/fi.h

```diff
@@ -209,8 +235,12 @@ int fi_rma_target_allowed(uint64_t caps);
         asm(".symver " #name "," #api "@@" DEFAULT_ABI)
 #else
 #  define symver(name, api, ver)
+#ifdef  HAVE_ALIAS_ATTRIBUTE
```

Comment:
Actually I take that back, I don't think the codebase supports compilers w/o alias support.